packages/cli/src/report/report-disk-writer.spec.ts

```diff
@@ -31,37 +32,67 @@ describe('ReportDiskWriter', () => {
         testSubject = new ReportDiskWriter(fsMock.object, pathMock.object);
     });
 
+    afterEach(() => {
+        fsMock.verifyAll();
```

Comment:
Not sure what the pattern is here but IIRC the pattern in the web project is to avoid verify in afterEach method.

Reply:
the pattern here is to have common mocks setup to verify so they are not missed or skip. Had several cases when this helped. It will not break the ability verify locally in test  It will also verify any setup set in a test or skip if it is not set in test.